test/TestExtensions/NuGet.StaFact/CIOnlyNuGetWpfTheoryAttribute.cs

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using NuGet.Test.Utility;
+using Xunit;
+using Xunit.Sdk;
+
+namespace NuGet.StaFact
+{
+    [AttributeUsage(AttributeTargets.Method, AllowMultiple = false)]
+    [XunitTestCaseDiscoverer("NuGet.StaFact.NuGetWpfTheoryDiscoverer", "NuGet.StaFact")]
+    public class CIOnlyNuGetWpfTheoryAttribute : TheoryAttribute
```

Comment:
you might want to extend from NuGetWpfTheoryAttribute

Reply:
@rohit21agrawal by doing this I can remove the `AttributeUsage` and `XunitTestCaseDiscoverer`, because they will be inherited, correct?

Reply:
depends on whether the attributes themselves have Inherited=true set

Reply:
I checked and they will, so removing them works perfectly.